docs/content/operations/multitenancy.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: doc_page
+---
+# Multitenancy Considerations
+
+Druid is often used to power user-facing data applications and has several features built in to better support high 
+volumes of concurrent queries.
+
+## Parallelization Model
+
+Druid's fundamental unit of computation is a [segment](../design/segments.html). Nodes scan segments in parallel and a 
+given node can scan `MIN(druid.processing.numThreads, Runtime.getRuntime().availableProcessors())` concurrently. To 
```

Comment:
what's the MIN there for? I think this would be more clear and accurate if it just said "druid.processing.numThreads"